client/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release Notes
 
```

Comment:
So, now, can we add the placeholder for the next version?

Reply:
xD
![image](https://user-images.githubusercontent.com/15327919/53876442-12733680-4007-11e9-9144-9fbd589ccb3d.png)

Reply:
Re-crap. I thought I asked this question during the develop→master merge and you answered “to, it would end up in master”.